Extension/src/nativeStrings.json

```diff
@@ -248,7 +248,7 @@
     "memory_limit_shutting_down_intellisense": "Shutting down IntelliSense server: {0}. Memory usage is {1} MB and has exceeded the {2} MB limit.",
     "failed_to_query_for_standard_version": "Failed to query compiler at path \"{0}\" for default standard versions.  Compiler querying is disabled for this compiler.",
     "unrecognized_language_standard_version": "Compiler query returned an unrecognized language standard version.  The latest supported version will be used instead.",
-    "intellisense_process_crash_detected": "IntelliSense process crash detected.",
+    "intellisense_process_crash_detected": "IntelliSense process crash detected. {0}",
```

Comment:
I'm a little worried about this. It'll have something like "handle_completion" at the end, without any connection to the previous sentence. Can you just have 2 strings, the 2nd saying "IntelliSense process crashed while processing '{0}'"?

Reply:
I think we should leave it as-is. Maybe put it in parentheses since it's not something the customer should really care about.

Reply:
Yeah, but if it says "IntelliSense processed crash" then customers are going to care about the crash. I'd prefer that we not have to give repeated explanations in GitHub issues about what that additional "handle_completion" text is.

Reply:
Then I vote to leave it out.  They're not going to know what `handle_completion` means either way.

Reply:
@bobbrow We will when they give us the logging -- e.g. "IntelliSense crashed while processing 'extract_function'" after the user selects some code. Otherwise, the logging will have a lot of messages and it will be hard to tell which IntelliSense operation is the cause.

Reply:
I talked to Tarik -- I'd be fine with the `{0}` being replaced by `.` or `: <function_name>`.

Reply:
Sean and I discussed a compromise.